skfuzzy/control/tests/test_controlsystem.py

```diff
@@ -53,7 +53,7 @@ def test_tipping_problem():
     tip_sim2.compute()
 
     assert tip_sim.output == tip_sim2.output
-    tst.assert_allclose(tip_sim.output['tip'], 20.244508118433625)
+    tst.assert_allclose(tip_sim.output['tip'], 19.847607362)
```

Comment:
I suppose the argument against would be, simply, that the discretization error in the prior method amounted to fully 7% error. I'll try running this example with very finely spaced universes and see if it approaches 19.847.